.goreleaser.yml

```diff
@@ -1,3 +1,6 @@
+# yaml-language-server: $schema=https://goreleaser.com/static/schema.json
```

Comment:
While we're at it, should we pin the schema version as instructed here:https://goreleaser.com/customization/#pin-the-schema-version?

Reply:
Done